packages/playground/website-extras/src/php-playground/constants.ts

```diff
@@ -0,0 +1,23 @@
+export const DEFAULT_PHP_VERSION = '8.4';
+export const DEFAULT_WP_VERSION = '6.8';
+// @TODO: Use URL imported from vite build-time config
+export const DEFAULT_WP_REMOTE = 'http://127.0.0.1:5400/remote.html';
+// export const DEFAULT_WP_REMOTE = 'https://playground.wordpress.net/remote.html';
```

Comment:
I think this is the reason for the error on https://playground.wordpress.net/beta-php-playground.html
<img width="1565" height="925" alt="Screenshot 2025-10-01 at 17 05 26" src="https://github.com/user-attachments/assets/bc6cdae8-88f0-4c95-85c3-ad3ab105a130" />

Reply:
yup, I'm adjusting this at https://github.com/WordPress/wordpress-playground/pull/2717. We'll need some e2e tests, too, to make sure it keeps working